### DISCUSSION TITLE
Set scopes for Google Provider on signIn in Next Auth 4

### DISCUSSION THREAD

**Discussion by huksley** [nextauthjs/next-auth#4557] [Help] [answered]
In the previous version, there was undocumented `scope?: string` option to `Providers.Google` configuration.

In Next Auth 4 it is missing in the source code.
Is there any way to set additional scopes, besides `openid email profile`

https://github.com/nextauthjs/next-auth/blob/main/packages/next-auth/src/providers/google.ts

**Comment by huksley** [comment]
I think I found in the docs, is that right? 

https://next-auth.js.org/configuration/providers/oauth#authorization-option

```js
authorization: {
  url: "https://example.com/oauth/authorization",
  params: { scope: "email" }
}
```

**Comment by feri-irawan** [comment, marked as answer]
You can do like this:
```ts
authorization: {
params: {
  scope: [
    "openid",
    "https://www.googleapis.com/auth/userinfo.email",
    "https://www.googleapis.com/auth/userinfo.profile",
    // and more scope urls
  ].join(" "),
  prompt: "consent",
  access_type: "offline",
  response_type: "code",
},
},
  ```

**Reply by Manikantaanyam** [reply]
hmm what if i need to add additional scopes later in project. How to do it i mean the incremental authorization ....
